Forcepoint Email Security Mapping (#25368)
demisto/content: Packs/ForcepointEmailSecurity/README.md

# Forcepoint Email Security
This pack includes Cortex XSIAM content. 
## Configuration on Server Side
You need to configure Forcepoint Email Security to forward Syslog messages in CEF format.

From the Forcepoint Security Manager, follow these instructions;
1. Navigate to the page **Settings** > **General** > **SIEM Integration**.

> 2. Mark the check box **Enable SIEM integration for all email appliances**

Reviewer: ```suggestion
2. Select the checkbox **Enable SIEM integration for all email appliances**.
```

3. Configure the following SIEM settings;

>    3.1. At **IP address or hostname**, enter the relevant IP address or hostname.

Reviewer: ```suggestion
   a. At **IP address or hostname**, enter the relevant IP address or hostname.
```

>    3.2. At **Port**, enter the relevant Port number.

Reviewer: ```suggestion
   b. At **Port**, enter the relevant Port number.
```

>    3.3. From the **Transport protocol** section, enter the relevant protocol (**UDP**).

Reviewer: ```suggestion
   c. From the **Transport protocol** section, enter the relevant protocol (**UDP**).
```

>    3.4. From the **SIEM format** pull-down menu, enter the relevant format (**syslog/CEF**).

Reviewer: ```suggestion
   d. From the **SIEM format** dropdown menu, enter the relevant format (**syslog/CEF**).
```

>    3.5. Confirm configuration by clicking on **Send Test Message**

Reviewer: ```suggestion
   e. Confirm configuration by clicking **Send Test Message**.
```

>    * Click on **OK** once you are done.

Reviewer: ```suggestion
4. Click **OK** once you are done.
```
